data-mdj/2017-2018-randomized-alternating-nonmatching/1ew-dockets-used.txt

```diff
@@ -0,0 +1,194 @@
+1561924964288_MJ-05205-CR-0000418-2018_01_17_12_18_namei_3_page_1
+1561924969580_MJ-10208-CR-0000245-2018_01_17_12_18_namei_3_page_1
+1561924974103_MJ-10104-CR-0000188-2018_01_17_12_18_namei_3_page_1
+1561924977502_MJ-44301-CR-0000039-2018_01_17_12_18_namei_3_page_11561925834837_MJ-32248-CR-0000456-2018_01_17_12_18_namei_10_page_1
```

Comment:
Do you know what happened here?

Reply:
Yeah, I fixed it. The above lines were from before I changed it to add a newline after each entry, and I forgot the newline after the last one when manually fixing it.